pages/windows/remove-appxpackage.md

```diff
@@ -1,20 +1,20 @@
-# remove-appxpackage
+# Remove-AppxPackage
 
 > A PowerShell utility to remove an app package from one or more user accounts.
-> More information: <https://learn.microsoft.com/powershell/module/appx/remove-appxpackage>.
+> More information: <https://learn.microsoft.com/powershell/module/appx/Remove-AppxPackage>.
```

Comment:
Should the link be capitalized too?

Reply:
Still redirected to the non-capitalized version.